### DISCUSSION TITLE
something like tempperery channel operator same for administrater

### DISCUSSION THREAD

**Discussion by amirmahdifard** [BearWare/TeamTalk5#1439] [General]
### Description

so you see that how you put an operator password for your channel, and if anyone have that password, they can make they selves channel operator but not permanently and just until they log off from the server, make the same for making a temperary admin: as this: you should bee able to set a password for that, and if anyone have that password, they could make they selves or other people as an administrater by entering that password.

### Application

- [X] qtTeamTalk
- [X] TeamTalkAndroid
- [X] iTeamTalk
- [X] TeamTalkServer

**Comment by amirmahdifard** [comment]
guys, what is the meaning of my eshoo mooved to discotion? means that this suggestion won't be happen? or what? please explane to me if possible. thanks.

**Comment by HDKESL** [comment]
snx for this discation
